CMakeLists.txt

```diff
@@ -60,7 +62,7 @@ add_definitions(${DBusCpp_CFLAGS_OTHER})
 add_definitions(${Glibmm_CFLAGS_OTHER})
 add_definitions(${LXC_CFLAGS_OTHER})
 add_definitions(${Jansson_CFLAGS_OTHER})
-add_definitions(-DINSTALL_PREFIX="${CMAKE_INSTALL_PREFIX}")
+add_definitions(-DINSTALL_PREFIX="${CMAKE_INSTALL_FULL_BINDIR}")
```

Comment:
If this is already the bindir, why do we add /bin to the setup_softwarecontainer.sh script? Also, it feels a bit misleading to say install prefix, when it is actually the bindir we're adding here.